### PR TITLE
Awesome book with ES6

### PR DESCRIPTION
**Pull request by sadaf-Daneshgar**
**New Feature**

- Add Html file
- Add CSS file
- Add JavaScript file 
- Add modules
- Configure the linter
- Add the luxon package

### CODE REVIEW

File: modules/addbook.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    });
+  }
+
+  displayBooks() {
+    // This method will generate the complete booklist and display it
+    const bookList = document.querySelector(this.elemToInsertData);
+    bookList.innerHTML = this.generateHtmlForBookList();
+    this.activateRemoveButton();
+    this.formInputFieldNormalize();
+  }
```
Comment (LionRouge1) [inline modules/addbook.js]: - [ ] You have done a great job displaying the book :clap: However the books do not display when reload the page. Kindly make sure the books saved in the localstore are display when the pages loaded.

**HIT:**
```js
document.addEventListener("DOMContentLoaded", () => {
  // your code here
});
```

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @sadaf-Daneshgar, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job creating all the requested file :+1: 
- Nice work dividing you code into modules :1st_place_medal:  
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL]
- [ ] Please do not add change to a PR after submit it for review. Kindly make sure you are done with everything before submit for review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/addbook.js (hunk 1 above)
Comment (sadaf-Daneshgar) [reply modules/addbook.js]: Hello @LionRouge1 
It means I should put my code inside of this??

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @sadaf-Daneshgar,

✔️ Books can be added and removed

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**